tests/Affects/Mails/Unit/ConfigureMailsTest.php

```diff
@@ -16,13 +16,49 @@
 
 namespace Tenancy\Tests\Affects\Mails\Unit;
 
+use Illuminate\Mail\Transport\ArrayTransport;
+use Mockery;
 use Tenancy\Affects\Mails\Events\ConfigureMails;
 use Tenancy\Affects\Mails\Provider;
+use Tenancy\Facades\Tenancy;
 use Tenancy\Tests\Affects\AffectsEventUnitTestCase;
 
 class ConfigureMailsTest extends AffectsEventUnitTestCase
```

Comment:
One test I would really like to see is if the `alwaysFrom` functionality of the Mailer works per tenant. I think it's the most common use case for what people will actually do with the mailer and would be really nice to have.